README.md

```diff
@@ -76,9 +80,9 @@ Find more install options on [Apple Silicon](https://docs.jina.ai/get-started/in
 
 Document, Executor and Flow are three fundamental concepts in Jina.
 
-- [**Document**](https://docarray.jina.ai/) is the fundamental data structure.
-- [**Executor**](https://docs.jina.ai/concepts/executor/) is a Python class with functions that use Documents as IO.
-- [**Flow**](https://docs.jina.ai/concepts/flow/) ties Executors together into a pipeline and exposes it with an API gateway.
+- [**Document**](https://docarray.jina.ai/) from [DocArray](https://github.com/docarray/docarray) is the fundamental data structure behind data validation and serialization.
+- [**Executor**](https://docs.jina.ai/concepts/executor/) is a Python class that can serve logic using Documents.
```

Comment:
I like better old definition perhaps

Reply:
transforms and processes Documents

Reply:
Same here. That sounds more straightforward